src/functions/getJournal/application/context.ts

```diff
@@ -0,0 +1,14 @@
+import 'reflect-metadata';
```

Comment:
I think this file should be part of framework

Reply:
I'd have thought the same, but interestingly it breaks without this import.

Reply:
The whole file I mean (couldn't put a comment the line above!)

Reply:
Ah I see, you're referring to moving it to the `framework` directory. I'm still trying to understand the distinction between `framework`, `application` and `domain`. What are your thoughts as to why this file should move?

Reply:
Moved.